arrow-fx-coroutines-test/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm"
+    id "org.jlleitschuh.gradle.ktlint"
+}
+
+apply plugin: 'kotlinx-atomicfu'
+
+apply from: "$SUB_PROJECT"
+apply from: "$DOC_CREATION"
+
+dependencies {
+    api "io.arrow-kt:arrow-core:$VERSION_NAME"
+    api project(":arrow-fx-coroutines")
+    api project(":arrow-fx-coroutines-stream")
```

Comment:
Should this not include `arrow-core-test`? Some of the generators defined below are included there iirc

Reply:
`arrow-core-test` and co are still behind on KotlinTest :(